### PR TITLE
[SHARE-377][Improvement] Update read the docs

### PR DESCRIPTION
**Pull request by leb2dg**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -10,18 +10,18 @@ SHARE harvests metadata nightly from 100+ repositories, transforms that metadata
...
    - Thorough search of creative works:
         + Creative works: `</api/v2/search/creativeworks/_search_>`_
         + more info on the `elasticsearch docs`_ page
    - Data aggregations across fields
-- A python library called ``sharepa`` for SHARE Parsing and Analysis
 - An `Ember application`_ using the SHARE API for:
    - Discovering new Projects
    - Curation
    - Corrections and Updates
+- A python library called ``sharepa`` for SHARE Parsing and Analysis
```
Comment (aaxelb) [inline docs/index.rst]: Does sharepa still work as expected? Does it need to be updated to match changes in the past X months?

Comment (leb2dg) [inline docs/index.rst]: Nope, but that is definitely a different ticket. There is a PR on sharepa for a needed update. (I did not add a link to sharepa from the docs it already existed in multiple places)

Comment (aaxelb) [inline docs/index.rst]: Cool cool, just curious

**Comment by leb2dg** [comment]
Good catch, I'll update the case insensitive stuff.

**Pull request merged**